packages/tasit-contract-based-account/src/GnosisSafe.js

```diff
@@ -229,10 +223,11 @@ export default class GnosisSafe extends Contract {
       gasPrice,
       gasToken,
       refundReceiver,
-      signatures,
-      gasParams
+      signatures
     );
 
-    return execTxAction.getTransaction();
+    const rawTx = await execTxAction._toRaw();
```

Comment:
Yeah, I think maybe _toRaw could just be defined for `tasit-contract-based-account`

Reply:
Any idea how I can do that?
`GnosisSafe` extends `Contract`, `Action` is a separated object.

Reply:
Good point. I guess the underscore does a good enough job of showing that it only will be used internally

Reply:
Oh, maybe there could be something separate in `tasit-contract-based-account` that extends `Action` and adds this functionality? Requires more thought on my end, though.

Reply:
Since es6 hasn't properly protected methods, I think that will be almost the same, we'll need to expose the `rawTx` field (maybe as _rawTx) instead of having that thru the function.

Reply:
Ah yeah - never mind! Getter function is better

Reply:
Okay, could I resolve this?

Reply:
Yep. I'm resolving it now.